flutter/shell/platform/tizen/channels/platform_channel.cc

```diff
@@ -138,6 +154,35 @@ void PlatformChannel::HandleMethodCall(
     document.AddMember(rapidjson::Value(kValueKey, allocator),
                        rapidjson::Value(ClipboardHasStrings()), allocator);
     result->Success(document);
+  } else if (method == kExitApplicationMethod) {
+    const rapidjson::Value& document = *arguments;
+    rapidjson::Value::ConstMemberIterator iter =
+        document.FindMember(kExitTypeKey);
+    if (iter == document.MemberEnd()) {
+      result->Error(kExitRequestError, kInvalidExitRequestMessage);
```

Comment:
nit: You might inline `kInvalidExitRequestMessage` for consistency. (See other occurrences of `result->Error()` in this file.)
```suggestion
      result->Error(kExitRequestError, "Invalid application exit request.");
```

Reply:
Removed kInvalidExitRequestMessage.